api/HTMLButtonElement.json

```diff
@@ -654,10 +654,10 @@
               "version_added": "27"
             },
             "safari": {
-              "version_added": "10"
+              "version_added": "10.1"
```

Comment:
I don't trust these changes unless we've tested both Safari 10 and 10.1. Does the updater script set a ≤ 10.1 range here, or might we be setting the wrong version in lots of places without any indication that the underlying data is incomplete?

Reply:
The collector tests both 10 and 10.1; otherwise, it would set a range to ≤10.1, which would cause no modifications to the data.

Reply:
Sweet!